doc/user-manual/language/module-system.lagda.rst

```diff
@@ -299,3 +299,13 @@ to make the type checker able to figure out that you wanted the natural number s
 
 Also record declarations define a corresponding module, see
 :ref:`record-modules`.
+
+References
+----------
+
+The initial design of Agda 2 module system is covered
+`in Ulf Norell thesis <https://www.cse.chalmers.se/~ulfn/papers/thesis.pdf>`_.
```

Comment:
Usually, the prepositions are not part of a citation link, cf. the `\citeauthoryear` command in LaTeX.
(2 more of these)

Reply:
You mean like "in `Ulf Norell thesis ...`"? Then fixed.